docker-compose.yml

```diff
@@ -5,7 +5,7 @@ redis:
 db:
   image: orchardup/postgresql
   ports:
-    - 5432:5432
+    - 4040:5432
```

Comment:
Please change back in both PRs.

Reply:
Thank you noted, the difference is **iss710** has tests and fix for the ResetPassword component and **iss715**  has test and fix for the FormDate component